examples/bpf/dropper.c

```diff
@@ -0,0 +1,83 @@
+#include <stdint.h>
```

Comment:
Should we rename this file to `stateless-firewall.c` and the other one from `sffw.c` to `stateful-firewall.c`? `dropper` sounds a bit too generic.